install.sh

```diff
@@ -85,55 +170,104 @@ if !([ -f /etc/ros/rosdep/sources.list.d/20-default.list ]); then
-if (env | grep --quiet "SEMAPHORE=true"); then
-    mv ~/Sub8 "$CATKIN_DIR/src"
-fi
+CMAKE_PREFIX_PATH=$CMAKE_PREFIX_PATH:/opt/ros/kinetic
+git clone https://github.com/ros-drivers/driver_common
+catkin_make -C $CATKIN_DIR -B
 
-# Sources the workspace's configurations for bash on this user account
-source "$CATKIN_DIR/devel/setup.bash"
-if !(cat ~/.bashrc | grep --quiet "source $CATKIN_DIR/devel/setup.bash"); then
-    echo "source $CATKIN_DIR/devel/setup.bash" >> ~/.bashrc
-fi
+source ../devel/setup.bash
 
-# Check if the sub is set up; if it isn't, set it up
-if !(ls "$CATKIN_DIR/src" | grep --quiet "Sub8"); then
-    instlog "Looks like you don't have the sub set up, let's do that"
-    cd "$CATKIN_DIR/src"
-    git clone -q https://github.com/uf-mil/Sub8.git
-    cd Sub8
+# Clones the Sub8 directory unless it already exists
+if !(ls $CATKIN_DIR/src | grep --quiet "Sub8"); then
+    instlog "Downloading the Sub8 repository"
+    cd $CATKIN_DIR/src
+    git clone https://github.com/uf-mil/Sub8.git
+    cd $CATKIN_DIR/src/Sub8
     git remote rename origin upstream
-    instlog "Make sure you change your git to point to your own fork! (git remote add origin your_forks_url)"
+    instlog "Make sure you change your git origin to point to your own fork! (git remote add origin your_forks_url)"
 fi
 
-# Update the hosts file if necessary
-if (env | grep --quiet "SUB=true"); then
-    $CATKIN_DIR/src/Sub8/scripts/update-hosts.bash
+# Adds Sub8 aliases to $BASHRC_FILE unless already exists
+if !(cat $BASHRC_FILE | grep --quiet "source $ALIASES_FILE"); then
+    echo "" >> $BASHRC_FILE
+    echo "# Adds Sub8 aliases to shell environment" >> $BASHRC_FILE
+    echo "source $ALIASES_FILE" >> $BASHRC_FILE
 fi
 
-# Install external dependencies with another script
-instlog "Running the get_dependencies.sh script to update external dependencies"
+#================================#
+#    Dependency Installation     #
+#================================#
+instlog "Installing Sub8 dependencies"
+
+sudo apt-get install python-scipy
+sudo apt-get install tesseract-ocr
+sudo apt-get install libusb-1.0-0-dev
+
+# Tools
+sudo apt-get install -qq sshfs
+sudo apt-get install -qq git-lfs gitk
+git lfs install --skip-smudge
+sudo apt-get install -qq tmux
+
+# Libraries needed by txros
+sudo apt-get install -qq python-twisted socat
+
+# Package management
+sudo pip install -q -U setuptools
+
+# Service identity verification
+sudo pip install -q -U service_identity
+
+# Utilities
+sudo pip install -q -U argcomplete
+sudo pip install -q -U tqdm
+sudo pip install -q -U pyasn1
+sudo pip install -q -U characteristic
+sudo pip install -q -U progressbar
+
+# Machine Learning
+sudo pip install -q -U scikit-learn > /dev/null 2>&1
+
+# Visualization
+sudo pip install -q -U mayavi > /dev/null 2>&1
+
+instlog "Cloning common GIT repositories"
 cd $CATKIN_DIR/src
-$CATKIN_DIR/src/Sub8/scripts/get_dependencies.sh
+git clone https://github.com/RustyBamboo/camera1394.git
```

Comment:
http://wiki.ros.org/camera1394... There doesn't seem to be support for kinetic...